public/index.html

```diff
@@ -18,7 +18,7 @@ <h5>ALPHA</h5>
     </header>
     <div id="game">
       <div class="centergroup">
-        <label>Name</label><input type="text" id="name" /><br />
+        <label>Name:</label><input type="text" id="name" /><br />
```

Comment:
Place these tags on separate lines
```suggestion
        <label>Name:</label>
        <input type="text" id="name" />
        <br />
```